media/css/products/relay/landing.scss

```diff
@@ -41,19 +52,45 @@ $brand-theme: 'firefox';
     }
 }
 
-.c-brands {
+.c-hero-title {
+    @include text-title-md;
+}
+
+.c-hero-body {
+    @include text-body-lg;
+}
+
+.c-hero-brands {
+    @include text-body-sm;
     border-bottom: 1px solid $color-marketing-gray-20;
     border-top: 1px solid $color-marketing-gray-20;
```

Comment:
**Nit:** The borders seem _very_ faint. Maybe we can darken them to `$color-marketing-gray-30`?

Reply:
Emdarkened.